pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1654,12 +1685,14 @@ def _populate_right_side(self, selector):
         # new devices that are not btrfs subvolumes.
         # Do this after the device type combo is set since
         # on_device_type_changed doesn't account for device existence.
-        fancy_set_sensitive(self._sizeEntry, device.resizable or (not device.exists and device.format.type != "btrfs"))
+        fancy_set_sensitive(self._sizeEntry, device.resizable or (
+                not device.exists and device.format.type != "btrfs"))
 
         if self._sizeEntry.get_sensitive():
             self._sizeEntry.props.has_tooltip = False
         elif device.format.type == "btrfs":
-            self._sizeEntry.set_tooltip_text(_("The space available to this mount point can be changed by modifying the volume below."))
+            self._sizeEntry.set_tooltip_text(_(
+                "The space available to this mount point can be changed by modifying the volume below."))
```

Comment:
This is over character limit. Please wrap the text.

Reply:
Fixed.